ext/hal/nordic/nrfx_config_nrf9160.h

```diff
@@ -193,6 +193,11 @@
         ((NRF_UARTE_Type *)DT_NORDIC_NRF_UARTE_UART_3_BASE_ADDRESS)
 #endif
 
+#ifdef DT_INST_0_NORDIC_NRF_UICR_BASE_ADDRESS
+#define NRF_UICR \
+        ((NRF_UICR_Type *)DT_INST_0_NORDIC_NRF_UICR_BASE_ADDRESS)
```

Comment:
The rest of this file doesn't use DT_INST macros. Should this be `DT_NORDIC_NRF_UICR_0_BASE_ADDRESS` instead?

Reply:
So, single-instance peripherals do not need a dts "alias" definition, and then DTC will generate a macro of _this_ format, i.e. DT_INST_0_NORD_NRF_xxxx_BASE_ADDRESS.

There are some other single-instance peripherals with the same format in this file.